style.css

```diff
@@ -784,3 +784,808 @@ textarea {
   margin-left: 302px;
   margin-top: -24px;
 }
+
+.horizontal_line,
+.desk_textcontainer,
+.desk_contents_section1,
+.desk_projects-container,
+.desk_projects-container1,
+.desk_projects-container2,
+.desk_projects-container3,
+.desk_languages-container,
+.contactform_end {
+  display: none;
+}
+
+/* starting with responsive web design */
+@media screen and (min-width: 600px) {
```

Comment:
You have used ` @media queries` for the Desktop breakpoint in the CSS file, Great Job! 👏 

- [ ] Make sure the value for **min-width** property should be in range of `992px - 1100px`

Reply:
I have changed this media screen min-width.